CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Airbrake Changelog
 
 ### master
 
+* **IMPORTANT:** added Shoryuken integration
+  ([#669](https://github.com/airbrake/airbrake/pull/669))
```

Comment:
@kyrylo sure. I'm using it from master now, do you see any possible issues with that?